packages/destination-actions/src/destinations/heap/trackEvent/index.ts

```diff
@@ -11,7 +11,7 @@ type HeapEvent = {
   app_id: string
   identity?: string
   user_id?: number
-  event: string
+  event: string | undefined
```

Comment:
I had to do this because of type, but theoretically this should never end up undefined